app/Http/Resources/V1/DefaultSsoSettingsResource.php

```diff
@@ -27,6 +27,7 @@ public function toArray($request)
             'guid' => $this->guid,
             'published' => $this->published,
             'organization' =>  $this->organization,
+            'role' =>  $this->role,
```

Comment:
Do we need to update JSON response for action using DefaultSsoSettingsResource.php ?